topics/dev/tutorials/bioblend-api/slides.html

```diff
@@ -1,26 +1,23 @@
 ---
 layout: tutorial_slides
 logo: "GTN"
+video: true
 
 title: "Scripting Galaxy using the API and BioBlend"
 questions:
-  - "What is a RESTful API?"
-  - "How to interact with Galaxy in a script?"
-  - "Why and when should I use BioBlend?"
-objectives:
-  - "Describe the principles of RESTful APIs"
-  - "Select the most efficient way to work with Galaxy"
-  - "Use the API and BioBlend to interact with Galaxy"
-requirements:
-  -
-    title: "IPython"
-    type: "external"
-    link: "https://ipython.org"
+  - What is a REST API?
+  - How to interact with Galaxy programmatically?
+  - Why and when should I use BioBlend?
+#requirements:
+#  -
+#    title: "IPython"
+#    type: "external"
+#    link: "https://ipython.org"
```

Comment:
I had to comment out the requirements here because the corresponding slide was without audio/subtitles, therefore messing up the audio sync with all following slides.

Reply:
Thanks for letting me know! I'll get that fixed so we can uncomment this again, I'm surprised nothing else triggered that bug!